recipes/default.rb

```diff
@@ -19,8 +19,10 @@
 
 influxdb_config node['influxdb']['config_file_path'] do
   config node['influxdb']['config']
+  notifies :restart, 'service[influxdb]', :delayed
 end
 
 service 'influxdb' do
-  action [:enable, :start]
+  supports :restart => true
```

Comment:
This got a syntax error.

```
RuboCop failed!
Running RuboCop...
Inspecting 27 files
...............C...........

Offenses:

recipes/default.rb:26:12: C: Use the new Ruby 1.9 hash syntax.
  supports :restart => true
           ^^^^^^^^^^^
```

I think this would work:

```
  supports { restart: true }
```